src/worker.py

```
@@ -1296,6 +1315,111 @@ async def _dispatch(request, env):
+            "parent_id": r["parent_id"],
+            "created_at": r["created_at"],
+            "updated_at": r["updated_at"],
+            "user_id": r["user_id"],
+            "author": await decrypt_aes(r["author_name"] or "", enc_key),
+        }
+        for r in (rows.results or [])
+    ]
+    return ok(comments)
+
+
+async def api_post_comment(req, env, activity_id: str, user, enc_key: str):
```

Ruff (0.15.7)

[warning] Too many return statements (8 > 6)

(PLR0911)